gem5-configs/configs-gapbs-tests/gapbs_config.py

```diff
@@ -130,4 +129,25 @@ def writeBenchScript(dir, benchmark_name, size, synthetic):
     print("Running the simulation")
     exit_event = m5.simulate()
     if exit_event.getCause() == "work started count reach":
-        print("done")
+        m5.stats.reset()
+        start_tick = m5.curTick()
+        start_insts = system.totalInsts()
+        # switching to atomic cpu if argument cpu == atomic
+        if cpu_type != 'kvm':
+            system.switchCpus(system.cpu, system.timingCpu)
+            print("Switch to detailed cpu model")
+    else:
+        print("ROI is not annotated!")
+        print('Exiting @ tick {} because {}'
+        .format(m5.curTick(), exit_event.getCause()))
```

Comment:
This line should be indented. Anytime a statement extends over a line, it should be clear for the reader that the next line is part of the previous statement.

```suggestion
            .format(m5.curTick(), exit_event.getCause()))
```